<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>417</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Agenda</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="welcomewidget" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>471</width>
      <height>211</height>
     </rect>
    </property>
    <widget class="QLabel" name="wel_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Purisa</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Welcome </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Purisa</family>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Agenda</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>wel_label</zorder>
    <zorder>label_2</zorder>
    <zorder>tableView</zorder>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>9</y>
      <width>681</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="searchwidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>361</width>
      <height>211</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Time:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchBtn_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End TIme:</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="startDT_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="endDT_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="createmtwidget" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>361</width>
      <height>211</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>59</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Title:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Participator:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End Time:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mtTitle">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="crtMtBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Meeting</string>
     </property>
    </widget>
    <widget class="QComboBox" name="parComboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="startDT">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="endDT">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="searchwidget" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>361</width>
      <height>211</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="searchTitle">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>59</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Title:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchBtn">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="opTitle">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>tableView</zorder>
   <zorder>welcomewidget</zorder>
   <zorder>searchwidget_2</zorder>
   <zorder>createmtwidget</zorder>
   <zorder>searchwidget</zorder>
   <zorder>opTitle</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionHome"/>
    <addaction name="separator"/>
    <addaction name="actionLogout"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuList">
    <property name="title">
     <string>List</string>
    </property>
    <addaction name="actionList"/>
    <addaction name="separator"/>
    <addaction name="actionList_All_Meetings"/>
    <addaction name="actionList_All_Sponsor_Meetings"/>
    <addaction name="actionList_All_Participant_Meetings"/>
   </widget>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>Create</string>
    </property>
    <addaction name="actionCreate_A_Meeting"/>
   </widget>
   <widget class="QMenu" name="menuDelete">
    <property name="title">
     <string>Delete</string>
    </property>
    <addaction name="actionDelete_A_Meeting"/>
    <addaction name="actionDelete_All_Meetings"/>
   </widget>
   <widget class="QMenu" name="menuQuery">
    <property name="title">
     <string>Query</string>
    </property>
    <addaction name="actionQuery_Meeting_By_Title"/>
    <addaction name="actionQuery_Meeting_By_Time_Interval"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuList"/>
   <addaction name="menuCreate"/>
   <addaction name="menuDelete"/>
   <addaction name="menuQuery"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLogout">
   <property name="text">
    <string>Logout User</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete User</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionList">
   <property name="text">
    <string>List All User</string>
   </property>
  </action>
  <action name="actionList_All_Meetings">
   <property name="text">
    <string>List All Meetings</string>
   </property>
  </action>
  <action name="actionList_All_Sponsor_Meetings">
   <property name="text">
    <string>List All Sponsor Meetings</string>
   </property>
  </action>
  <action name="actionList_All_Participant_Meetings">
   <property name="text">
    <string>List All Participator Meetings</string>
   </property>
  </action>
  <action name="actionCreate_A_Meeting">
   <property name="text">
    <string>Create A Meeting</string>
   </property>
  </action>
  <action name="actionDelete_A_Meeting">
   <property name="text">
    <string>Delete A Meeting</string>
   </property>
  </action>
  <action name="actionDelete_All_Meetings">
   <property name="text">
    <string>Delete All Meetings</string>
   </property>
  </action>
  <action name="actionQuery_Meeting_By_Title">
   <property name="text">
    <string>Query Meeting By Title</string>
   </property>
  </action>
  <action name="actionQuery_Meeting_By_Time_Interval">
   <property name="text">
    <string>Query Meeting By Time Interval</string>
   </property>
  </action>
  <action name="actionGuide">
   <property name="text">
    <string>Guide</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
